### ISSUE TITLE
Schema Errors: Found 2 fields with name "translations" in object

### ISSUE THREAD

**Issue by npoch** [nuagedelait/sanity-pagebuilder#3]
<img width="1036" alt="image" src="https://github.com/nuagedelait/sanity-pagebuilder/assets/29647127/246965de-6852-4b45-b0e8-db0fd150826c">

I have not added any schemas to override the default schemas.

Version: Sanity 3.30.0
Plugin Version: 1.1.1

I have gotten it to load everything but when I create a document it doesn't show the structure or allow me to save any documents.
then I moved some things around and I get the errors above.

```
export default defineConfig([
  {
	  name: 'pb-prod',
	  title: 'Pagebuilder-Production',
    basePath: '/production',
	  projectId: '<projectID>',
	  dataset: 'production',
	  plugins: [
      // structureTool(),
      pagebuilderTool({
        addBlocksSchemas: [],
        addContentSchemas: [],
        addManagmentSchemas: [],
        api: 'sanity api version, default : `v2023-08-01`',
        // languages: [],
      }),
    ],
	  // schema: {
	  //   types: schemaTypes,
    // },
  },
  {
  name: 'pb-staging',
  title: 'Pagebuilder-Staging',
  projectId: '<projectID>',
  dataset: 'production',
  basePath: '/staging',
  plugins: [
    // structureTool(),
    pagebuilderTool({
      addBlocksSchemas: [],
      addContentSchemas: [],
      addManagmentSchemas: [],
      api: 'sanity api version, default : `v2023-08-01`',
      // languages: [],
    }),
  ],
  // schema: {
  //   types: schemaTypes,
  // },
},
])
```

There are errors in my terminal that mentioned i18n functions. I'm not sure where I can configure that.

**Comment by nuagedelait** [comment]
if you don't use languages, set the attribute to undefined and not an empty array
I pushed a fix on languages, I noticed that when you don't have any languages, in some areas, my code was trying to get the current lang from url.

**Comment by npoch** [comment]
This resolved it and seems to be resolved with the new release of 1.1.1.

Thank you again.
